### DISCUSSION TITLE
Loading file error

### DISCUSSION THREAD

**Discussion by khalil387** [mrdbourke/pytorch-deep-learning#1156] [Q&A]
when I use
 ***loaded_model_0.load_state_dict(MODEL_SAVE_PATH)***

It raises this error:
_TypeError: Expected state_dict to be dict-like, got <class 'pathlib.PosixPath'>._
Can you help me?

**Comment by singing-cat** [comment]
You might try `loaded_model_0.load_state_dict(torch.load(MODEL_SAVE_PATH))`.
